tools/roslaunch/src/roslaunch/xmlloader.py

```diff
@@ -289,10 +297,24 @@ def _arg_tag(self, tag, context, ros_config, verbose=True):
         Process an <arg> tag.
         """
         try:
+            
```

Comment:
Unrelated change. Should be be reverted.

Reply:
Reverted.